vendor/engines/secure_rooms/app/views/secure_rooms/occupancies/index.html.haml

```diff
@@ -22,3 +22,21 @@
           %td= occupancy.entry_at
 - else
   %p.notice= text("views.admin.secure_rooms.occupancies.index.none")
+
+- if @problem_occupancies.present?
```

Comment:
We should do these as separate tabs, like we do in reservations and orders. That means they should probably be different controller actions.